### ISSUE TITLE
The heavy use of "(commercial)" is confusing and seems unnnecessary

### ISSUE THREAD

**Issue by chaals** [eea-oasis/baseline-standard#119]
In e.g. §2.6 there are many statements like

> ... a (commercial) agreement and (commercial) document ...

"Commercial Document" is a defined term, so should be used as such rather than a parenthetical adjective which is just confusing. It would be nice if it were a link to the definition, too.

Meanwhile, it isn't clear why "agreement" is _perhaps_ qualified, as "(commercial)". Is there some reason non-commercial agreements cannot be transacted with baseline? Is there any value in the parenthetical addition, to justify the effort of having to try and understand it?

**Comment by humbitious** [comment]
@chaals, you make a very good point here, and we discussed this on the 2021 standards meetings extensively under Anais' direction.  The compromise was to add this section to cover specifically the case of legal contracts (because the technical requirements in that case are more stringent), but there is no question -- and your observation confirms this -- that a new or casual reader of the standard could read this section and conclude incorrectly that these requirements apply to all cases. This, and several other of your recent observations, are something I'd like us to visit on the 2022 round of revisions (once the standard has been approved) on the 2022 Theme: Simple is Beautiful.  (That's the job of 2022...make everything about baselining simpler and more accessible/adoptable.)

**Comment by Therecanbeonlyone1969** [comment]
@chaals ... per decision in the WG meeting, please, respond to the above response. Otherwise, the issue will be closed at the next WG meeting.

**Comment by chaals** [comment]
If a Commercial Document is a type of Document (as you define them) then anything that applies to a Document applies to a Commercial Document. Which means you can say "Document" and it's clear.

If people are misreading and misunderstanding, there is real work to do on making sure we're agreeing on what we read, before we can decide if it's good enough for a release.

**Comment by Therecanbeonlyone1969** [comment]
@chaals ... point taken. Submitted PR #126 that removes (commercial) and (commercially) everywhere.

**Comment by atoulme** [comment]
I agree, upon reading the spec, that the mention of (commercial) is not needed. There are also instances of `(business)` in the introduction that could be removed.
